docs/pages/database-access/guides/mongodb-atlas.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Database Access with MongoDB Atlas
+description: How to configure Teleport Database Access with MongoDB Atlas.
+---
+
+# MongoDB Atlas
+
+In this guide you will:
+
+1. Configure Teleport for accessing your MongoDB Atlas cluster.
+2. Configure self-managed X.509 authentication on your Atlas cluster.
+3. Connect to your Atlas cluster via Teleport.
+
+## Prerequisites
+
+- Teleport version `7.0` or newer.
+- [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) cluster.
+
+## Step 1/3. Configure Teleport
+
+### Setup Teleport Auth and Proxy services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+### Setup Teleport Database service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Start Teleport Database service:
+
+```bash
+teleport db start \
+  --token=/tmp/token \
+  --auth-server=teleport.example.com:3080 \
+  --name=mongodb-atlas \
+  --protocol=mongodb \
+  --uri=mongodb+srv://cluster0.abcde.mongodb.net \
+  --ca-cert=/path/to/letsencrypt/root-ca
+  --labels=env=dev
+```
+
+See below for details on `--uri` and `--ca-cert` flags.
+
+<Admonition type="note">
+  The `--auth-server` flag must point to the Teleport cluster's proxy endpoint
+  because database service always connects back to the cluster over a reverse
+  tunnel.
+</Admonition>
+
+<Admonition type="tip">
+  You can start Database service using configuration file instead of CLI flags.
+  See [YAML reference](../reference/configuration.mdx).
+</Admonition>
+
+#### Connection endpoint (`--uri`)
+
+To find out your Atlas cluster's connection endpoint for the URI, use the
+Connect dialog on the Database Deployments overview page:
+
+![Connect](../../../img/database-access/guides/atlas/atlas-connect-btn@2x.png)
+
+Go through the "Setup connection security" step and select "Connect with the
+MongoDB shell" to view the connection string:
+
+![Connection string](../../../img/database-access/guides/atlas/atlas-connect@2x.png)
+
+Use only the scheme and hostname parts of the connection string in the URI:
+
+```txt
+--uri=mongodb+srv://cluster0.abcde.mongodb.net
+```
+
+#### Atlas CA certificate (`--ca-cert`)
+
+MongoDB Atlas uses certificates signed by Let's Encrypt as described in
+[Which certificate authority signs MongoDB Atlas cluster TLS certificates?](https://docs.atlas.mongodb.com/reference/faq/security/#which-certificate-authority-signs-mongodb-atlas-cluster-tls-certificates-)
+
+Download Let's Encrypt [root certificate](https://letsencrypt.org/certs/isrgrootx1.pem.txt)
+and use it as a CA in the database service configuration:
+
+```txt
+--ca-cert=/tmp/isrgrootx1.pem
+```
```

Comment:
I've tried to set this with teleport.yaml and I get this error ` field ca-cert not found in type config.Database` 

My setup looks like this

```
...
            - name: "mongodb"
              protocol: "mongodb"
              uri: "mongodb+srv://atlas-mongo-teleport.f8rg9.mongodb.net"
              ca-cert: "/etc/letsencrypt/isrgrootx1.pem"
              static_labels:
                env: dev-1
                engine: 🍃🌍
```

Reply:
@benarent The yaml configuration field is called `ca_cert_file` (which is consistent naming with other services in yaml config). See the yaml reference here: https://goteleport.com/docs/database-access/reference/configuration/.

Reply:
Thanks Roman, could we add an inline reference on this page for people following this guide.

Reply:
Good idea, added.